phx/performances/performances_scraper.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import re
+from datetime import date, datetime
+
+import requests
+from athletes.models import Athlete
+from bs4 import BeautifulSoup
+from events.models import Event
+from performances.models import Performance
+
+logger = logging.getLogger(__name__)
+
+
+class PerformancesScraper:
+
+    ATHLETE_PROFILE_URL = "https://www.thepowerof10.info/athletes/" \
+                          "profile.aspx?athleteid={athlete_id}&viewby=date"
+
+    def __init__(self):
+        self.events = {}
+        self.performances = []
+        self.inactive_athletes = set[str]()
+
+    def find_performances(self, athlete: Athlete, since: date):
+        """
+        Search Power of 10 for all performances by the given athlete since the
+        given date
+
+        Returns the total number of performances found
+        """
+
+        if not athlete.power_of_10_id:
+            logger.warning(f"No Power of 10 ID for {athlete}")
+            return 0
+
+        (performances, events, active) = self._scrape_profile(athlete, since)
+
+        if not active:
+            self.inactive_athletes.add(athlete.power_of_10_id)
+            return 0
+        else:
+            self.events.update(events)
+            self.performances.extend(performances)
+            return len(performances)
+
+    def _scrape_profile(self, athlete: Athlete, since: date):
+        url = PerformancesScraper.ATHLETE_PROFILE_URL.format(
+            athlete_id=athlete.power_of_10_id)
+        logger.info("Scraping {url}".format(url=url))
+
+        page = requests.get(url)
```

Comment:
Should we add any error handling in case this doesn't respond with the expected data? (e.g. a network error) - will it retry, or just try again the next time?

Reply:
As things stand, I think the whole job will stop and it won't try again until the next day. Made an issue to look into further though: https://github.com/orangespaceman/phx/issues/91